test/rest.test.js

```diff
@@ -319,89 +319,34 @@ describe('strong-remoting-rest', function() {
       );
     });
 
-    it('should support cors', function(done) {
+    it('should reject cross-origin requests', function(done) {
       request(app).post(method.url)
         .set('Accept', 'application/json')
         .set('Content-Type', 'application/json')
         .set('Origin', 'http://localhost:3001')
         .send({ person: 'ABC' })
-        .expect('Access-Control-Allow-Origin', 'http://localhost:3001')
-        .expect('Access-Control-Allow-Credentials', 'true')
-        .expect(200, done);
-    });
-
-    it('should skip cors if origin is the same as the request url', function(done) {
-      var server = request(app).post(method.url);
-      var url = server.url.replace('/testClass/testMethod', '');
-      server
-        .set('Accept', 'application/json')
-        .set('Content-Type', 'application/json')
-        .set('Origin', url)
-        .send({ person: 'ABC' })
-        .end(function(err, res) {
-          assert(res.get('Access-Control-Allow-Origin') === undefined);
-          assert(res.get('Access-Control-Allow-Credentials') === undefined);
+        .expect(200, function(err, res) {
```

Comment:
200 may not be the right http status code, test should check for the appropriate error code